ansible/roles/software/ckan/security-fix/tasks/main.yml

```diff
@@ -10,3 +10,21 @@
   with_items:
     - /etc/ckan
     - "{{ project_source_new_code_path }}"
+
+- name: Fix 'Content-Cache' header
+  replace:
+    dest: "{{ project_source_new_code_path }}/src/ckan/ckan/lib/base.py"
+    regexp: ' = "private"'
```

Comment:
For major file updates that would be legit. I don't believe it's possible to introduce duplicates with `lineinfile` using `regexp: '^SomeSetting ' ` though